src/shared/components/mutationTable/column/AlterationOverlapColumnFormatter.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { Mutation, Sample } from 'cbioportal-ts-api-client';
+import {
+    ComparisonGroup,
+    getPatientIdentifiers,
+    getSampleIdentifiers,
+} from 'pages/groupComparison/GroupComparisonUtils';
+import { MiniOncoprint } from 'shared/components/miniOncoprint/MiniOncoprint';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import ComplexKeyMap from 'shared/lib/complexKeyDataStructures/ComplexKeyMap';
+import { ComparisonMutationsRow } from 'shared/model/ComparisonMutationsRow';
+import { AlterationOverlapOverlay } from './alterationOverlap/AlterationOverlapOverlay';
+
+export default class AlterationOverlapColumnFormatter {
```

Comment:
We probably don't need classes and static functions for this new column formatter and all other column formatters implemented in this PR. This was how we implemented all these column formatters long time ago, but I think a simpler functional approach should just work fine.